src/NodeAnalyzer/MethodArgumentAnalyzer.php

```diff
@@ -26,18 +27,36 @@ public function hasMethodSecondArgument(Node $node): bool
 
         if (count($node->args) < 2) {
             return false;
-        }
+    }
+
+return true;
+}
 
-        return true;
+public function isMethodFirstArgumentString(Node $node): bool
+{
+    if (! $this->hasMethodFirstArgument($node)) {
+        return false;
     }
 
-    public function isMethodFirstArgumentString(Node $node): bool
-    {
-        if (! $this->hasMethodFirstArgument($node)) {
+    /** @var MethodCall $node */
+    return $node->args[0]->value instanceof String_;
+}
+
+public function isMethodSecondArgumentNull(Node $node): bool
+{
+    if (! $this->hasMethodSecondArgument($node)) {
             return false;
         }
 
         /** @var MethodCall $node */
-        return $node->args[0]->value instanceof String_;
+        $value = $node->args[1]->value;
+        if (! $value instanceof ConstFetch) {
+            return false;
+        }
+
+        /** @var \PhpParser\Node\Name $nodeName */
+        $nodeName = $value->name;
```

Comment:
Strange, it didn't solve.

@TomasVotruba Any clue?

Reply:
It compares it to own php-parser of version 3 that doesn't have such method yet.